substratevm/src/com.oracle.svm.agent/src/com/oracle/svm/agent/Agent.java

```diff
@@ -345,27 +324,19 @@ public static int onLoad(JNIJavaVM vm, CCharPointer options, @SuppressWarnings("
             }
         }
 
-        check(jvmti.getFunctions().SetEventCallbacks().invoke(jvmti, callbacks, SizeOf.get(JvmtiEventCallbacks.class)));
-        UnmanagedMemory.free(callbacks);
-
-        check(jvmti.getFunctions().SetEventNotificationMode().invoke(jvmti, JVMTI_ENABLE, JVMTI_EVENT_VM_START, nullHandle()));
-        check(jvmti.getFunctions().SetEventNotificationMode().invoke(jvmti, JVMTI_ENABLE, JVMTI_EVENT_VM_INIT, nullHandle()));
-        check(jvmti.getFunctions().SetEventNotificationMode().invoke(jvmti, JVMTI_ENABLE, JVMTI_EVENT_VM_DEATH, nullHandle()));
-        check(jvmti.getFunctions().SetEventNotificationMode().invoke(jvmti, JVMTI_ENABLE, JVMTI_EVENT_THREAD_END, nullHandle()));
-
         setupExecutorServiceForPeriodicConfigurationCapture(configWritePeriod, configWritePeriodInitialDelay);
         return 0;
     }
 
-    private static int parseIntegerOrNegative(String number) {
+    private int parseIntegerOrNegative(String number) {
```

Comment:
Our style checking gate will likely complain that this could be declared `static`, other methods too.

Reply:
I will make them static again.